packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -18,6 +19,8 @@ export interface WizardStep {
   name: React.ReactNode;
   /** The component to render in the main body */
   component?: any;
+  /** The content to render in the drawer panel */
+  drawerPanelContent?: any;
```

Comment:
i've noticed that leaving this out of one of the steps appears to remove the drawer for that step, even when `hasDrawer` is true on the wizard! Not sure if its worth removing the drawer scaffolding in those cases so that drawers without drawerPanelContent defined for a step dont wrap the whole wizard in drawer components? It's probably not trivial, so as long as there are not interactive or style side effects to situations like that, then it's likely fine as is.